src/workspace-data/data-table/wds/attribute-utils.js

```diff
@@ -0,0 +1,30 @@
+import * as Utils from 'src/libs/utils';
+
+export const getAttributeType = (attributeName, entityTypeAttributes, attributeValue, dataProvider) => {
```

Comment:
I don't think we need the attribute value to get the type for WDS. It should be available from the schema.

Reply:
see comment https://github.com/DataBiosphere/terra-ui/pull/4588/files#r1472157907 on my thoughts on this 😄